### ISSUE TITLE
The default colour of #FD647A provides insufficient colour contrast over white

### ISSUE THREAD

**Issue by michael-milette** [trema-tech/moodle-theme_trema#72]
When text of colour #FD647A is displayed over a white background, the colour contrast fails to meet [WCAG 2.1 level AA 1.4.3 Minimum Contast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) requirements.

![image](https://user-images.githubusercontent.com/3808579/138742849-ba889b2e-fc58-447c-8d13-a9cac8842283.png)
![image](https://user-images.githubusercontent.com/3808579/138743081-d39f9507-1eaf-4a38-9277-2d2a9dc10f34.png)

Note when choosing a new colour that items with black text over the new colour may then fail colour contrast ratio requirements.

Suggestion: You may need to add additional Trema theme settings in order to accommodate possible dark and light colour backgrounds.

Best regards,

Michael Milette

**Comment by rmady** [comment]
Thanks for the advertisement @michael-milette. I'll add this new settings.

Best regards,
Rodrigo Mady
